packages/discovery-provider/src/api/v1/users.py

```diff
@@ -1968,7 +1968,6 @@ class ConnectedWallets(Resource):
         responses={200: "Success", 400: "Bad request", 500: "Server error"},
     )
     @ns.marshal_with(connected_wallets_response)
-    @cache(ttl_sec=10)
```

Comment:
why was this even here? afaik if you're authed this cache decorator does nothing anyway

Reply:
Not sure, think we just cache things by default? Def not busted by auth tho, server returned cached response during testing after making a removal or addition